refresh-data.sh

```diff
@@ -12,39 +12,43 @@ refresh_dump_name=$1
 
 download_data() {
 	echo 'Downloading fresh production Django database dump...'
-	prod_archive="$refresh_dump_name".gz
+	prod_archive="$refresh_dump_name"
 	curl -o $prod_archive $CFGOV_PROD_DB_LOCATION
-	gunzip -f $prod_archive
 }
 
 refresh_data(){
-	echo 'Dropping db'
```

Comment:
I should point out that this is also a change from the current behavior. During normal DB refresh it's difficult to actually delete and recreate the database in a generic way, without using the DB-specific commands (like Postgres `dropdb`). For this reason I've modified this script to just re-import the data from the dump and skip the drop/recreate. One side effect this should have is to potentially leave old tables around.